src/license_manager_agent_ops.py

```diff
@@ -35,11 +35,23 @@ def __init__(self, charm):
         self._charm = charm
 
     def setup_cache_dir(self):
-        cache_dir = Path("/var/cache/license-manager")
-        if not cache_dir.exists():
-            cache_dir.mkdir()
-            shutil.chown(cache_dir, user="slurm")
-            cache_dir.chmod(0o700)
+        """Set up cache dir."""
+        CACHE_DIR = Path("/var/cache/license-manager")
```

Comment:
Since the cache dir value is hardcoded here, why not declare it in the class, together with the other constants?

Reply:
`CACHE_DIR` is just used inside this method, so I see no problem to keep it as a local variable. We can move it to a class variable if it becomes needed in more places.